helm/openwhisk/values.yaml

```diff
@@ -139,9 +139,9 @@ whisk:
     includeSystemTests: false
   versions:
     openwhisk:
-      buildDate: "2022-06-23-06:18:00Z"
```

Comment:
I made the details of this timestamp up. Am I supposed to have copied it from somewhere?

Reply:
I believe @dgrove-oss has an idea.

Reply:
My habit has been to use the UTC time of the commit we are using in `gitTag` as the `buildDate`.  I don't think this actually matters, but it's what I'd been doing ;)